packages/code/src/Code/Code.styles.ts

```diff
@@ -112,6 +154,47 @@ export function getCodeWrapperVariantStyle(theme: Theme): string {
   `;
 }
 
+export const expandButtonStyle = css`
+  align-items: center;
+  border: none;
+  /**
+    Code wrapper's border radius is 12px. Matching that creates a very 
+    slight gap between the button and the code wrapper. Decreasing by
+    1px removes gap without making focused state look off.
+  */
+  border-radius: 0 0 11px 11px;
+  border-width: 1px 0 0 0;
+  border-style: solid;
+  display: flex;
+  font-family: ${fontFamilies.default};
+  font-size: ${typeScales.body1.fontSize}px;
+  gap: 4px;
```

Comment:
Whenever possible, can you use spacing tokens?

Reply:
Oh absolutely, will update shortly